template/testsample.template

```diff
@@ -0,0 +1,106 @@
+/*
+* ./test/SampleRealtimeTest.js
+*
+* Starter test file for RealtimeChangeHandler.js
+*/
+
+const expect = require('chai').expect;
+const fs = require('fs-extra');
+const RealtimeChangeHandler = require('../src/RealtimeChangeHandler');
+const util = require('../src/Utils');
+
+// Here is the sample data you can use to test your Util functions.
+// This data is also used to prototype your new lens.
+
+let sampleData = JSON.parse(fs.readFileSync('./test/weatherByCountry.json'));
```

Comment:
This should be the result of processNode(), to match what's being passed in to realtimeChangeHandler.handle() in main.